### PR TITLE
show notification if disabled packages are not installed

### PR DESCRIPTION
**Pull request by tyleri**
Here's how the notification would look:
image

I tried looking into ways I could make a button that would just install it for them when they clicked, but I don't think that's supported in the API.

### CODE REVIEW

**Comment by josephch405** [comment]
The small irk that I have with this is simply the fact that we're literally telling users to download packages that are supposed to be disabled in the project - which does not seem to be Harm's idea at all (the original intent was to install enabled packages, but there isn't really a true "enabled packages" setting). Is it possible for us to allow for such a setting in the configs (say a "core.enabledpackages" or a "atomic-management.enabledpackages if we wanna keep it local), intercept it, then warn the user?

Other than that, this pull seems fine - it does what it's supposed to do and warns the user about potentially missing packages in the disabled list.

**Comment by tyleri** [comment]
Yeah I was thinking about the same thing Jo... Definitely something we should talk to Harms about probably.

**Comment by djh329** [comment]
So the answer to this lies in the config file itself. Enabled packages are listed by trying to configure them. It is unintuitive to have an enabledPackages section because that’s not how all the other Atom config.cson stuff works and people will get confused. We can keep track of all the top level settings, and if something is not `core` or `editor` then we assume it’s a package. This way we do not need a set of all package names

**Comment by josephch405** [comment]
Anyone wanna review again?

File: test.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+a = 20
+for i in range(a):
+  print(a)
```
Comment (djh329) [inline test.py]: What is this?

**Comment by djh329** [comment]
You can def merge this into sprint_4_fixes. I didn't test it, but if you know it works, you can also merge it into master. The code looks fine and I think you handle the 3 cases appropriately. If someone else wants to take a test run of the code and see if the notifications seem sane, that would probably be good

**Pull request merged**